src/mlx/warnings.py

```diff
@@ -165,15 +188,19 @@ def warnings_wrapper(args):
     parser.add_argument('--minwarnings', type=int, required=False, default=0,
                         help='Minimum amount of warnings accepted')
     parser.add_argument('--version', action='version', version='%(prog)s {version}'.format(version=pkg_resources.require('mlx.warnings')[0].version))
-
+    parser.add_argument('--config', dest='configfile', action='store', required=False, help='Config file in JSON format provides toggle of checkers and their limits')
```

Comment:
can/should we make this one exclusive to the d/s/j checker enabling?

Reply:
Yes, that would be a suggestion indeed. Will see to it.